<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">–£–º–Ω—ã–π —Ö–æ–ª–æ–¥–∏–ª—å–Ω–∏–∫</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(105, 105, 105);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="logo">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="reload">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(243, 243, 244);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QToolBox" name="toolBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 146, 146);</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="InFreezer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>885</width>
        <height>404</height>
       </rect>
      </property>
      <attribute name="label">
       <string notr="true">–ü—Ä–æ–¥—É–∫—Ç—ã –≤ —Ö–æ–ª–æ–¥–∏–ª—å–Ω–∏–∫–µ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="InFreezerContentContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>859</width>
            <height>378</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="inFreezerLayout">
           <property name="topMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ToBuy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>885</width>
        <height>404</height>
       </rect>
      </property>
      <attribute name="label">
       <string>–ß—Ç–æ –Ω—É–∂–Ω–æ –∫—É–ø–∏—Ç—å</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="ListToBuyContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>859</width>
            <height>313</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="listtobuylayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QPushButton" name="edit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>261</width>
            <height>32</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å —Å–ø–∏—Å–æ–∫ –∞–≤—Ç–æ–∑–∞–∫—É–ø–∫–∏</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="search_page">
      <attribute name="label">
       <string>–ü–æ–∏—Å–∫</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>859</width>
            <height>313</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="searchlayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLineEdit" name="searchEditLine">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>301</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="searchButton">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>10</y>
            <width>64</width>
            <height>32</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>üîç</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
